src/cst1201/CharacterCounter.java

```diff
@@ -19,11 +19,11 @@ public static void main(String[] args) {
 
         Scanner scanner = new Scanner(System.in);
 
-        String inputString = scanner.nextLine();
+        String inputString = scanner.nextLine().toUpper();

```

Comment:
:-1: We don't want to let the user know that the input string has changed internally. This would be confusing to the user.